src/index.js

```diff
@@ -32,14 +32,39 @@ function sortGroups(selector) {
   });
 }
 
+/**
+ * Remove duplicated properties
+ * @param {Object} selector - postcss selector node
+ */
+function removeDuplicatedProperties(selector) {
+  let actualNodeIndex = selector.nodes.length - 1;
+  // Remove duplicated properties from bottom to top
+  while (actualNodeIndex >= 1) {
+    let beforeNodeIndex = actualNodeIndex - 1;
+    while (beforeNodeIndex >= 0) {
+      if (selector.nodes[actualNodeIndex].prop === selector.nodes[beforeNodeIndex].prop) {
+        selector.nodes[beforeNodeIndex].remove();
+        actualNodeIndex--;
+      }
+      beforeNodeIndex--;
+    }
+    actualNodeIndex--;
+  }
+}
+
 const uniformStyle = parser(
   (selector) => {
     normalizeAttributes(selector);
     sortGroups(selector);
   }
 );
 
-module.exports = postcss.plugin('postcss-combine-duplicated-selectors', () => {
+const defaultOptions = {
+  removeDuplicatedProperties: false
+};
+
+module.exports = postcss.plugin('postcss-combine-duplicated-selectors', (options = defaultOptions) => {
```

Comment:
I would lean using destructuring to access values from the options object.
``` js
(options = defaultOptions) => {
```
to
``` js
({removeDuplicatedProperties = true}) => {
```